app/javascript/components/student/editor/FileEditor.tsx

```diff
@@ -45,6 +57,41 @@ export const FileEditor = forwardRef<FileEditorHandle, FileEditorProps>(
           keybindings: [monacoEditor.KeyCode.F2],
           run: onRunTests,
         })
+
+        MonacoServices.install(editor)
+        const url = normalizeUrl('ws://localhost:7658/ruby')
+        const webSocket = new (ReconnectingWebSocket as any)(url, [], {
```

Comment:
The `ReconnectingWebsocket` constructor doesn't have any type definitions. Thus I just typed it to `any`.

Reply:
I think you can use the `@types/reconnectingwebsocket` [NPM package](`https://www.npmjs.com/package/@types/reconnectingwebsocket`) for that.

Reply:
Thanks :) I've done this now.